.github/workflows/unit_test.yaml

```diff
@@ -34,7 +34,9 @@ jobs:
         run: python -m pip install --upgrade pip
       - name: Install dependencies
         run: |
-          python -m pip install torch torchvision torchao
+          NIGHTLY_VERSION=dev20241121
+          python -m pip install torch=="2.6.0.${NIGHTLY_VERSION}" torchvision=="0.20.0.${NIGHTLY_VERSION}" --extra-index-url https://download.pytorch.org/whl/nightly/cpu
+          python -m pip install torchao
```

Comment:
I don't think we want to change the default behavior of our unit test workflow here. Can we figure out a way to decouple the export CI a little more?

Reply:
Can you say a bit more? Are you ok with a separate CI job running these tests with torch nightly installed?

Reply:
Added `Export` CI workflow.